router/router_test.go

```diff
@@ -313,3 +391,58 @@ func wsServer(t *testing.T) {
 	}
 	log.Println("Out of server")
 }
+
+func TestUnitGetRouterFromConfigService(t *testing.T) {
+	t.Log("Testing: Retrieving router port from the config service, after router service startup")
+	expectedValue := "5002"
+	response, err := http.DefaultClient.Get(ConfigServiceUrl +"/config-service/variable/router_http_port")
+	if err != nil {
+		t.Errorf("Error getting router_http_port from config service: %v", err)
+		return
+	}
+	defer response.Body.Close()
+	body, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		t.Errorf("Error reading response data")
+		return
+	}
+	if response.StatusCode != 200 {
+		t.Errorf("Error response from config service: StatusCode: %v, Body: %v", response.StatusCode, string(body[:]))
+		return
+	}
+	if string(body[:]) != expectedValue {
+		t.Errorf("Expected value in response to be %v, instead %v", expectedValue, string(body[:]))
+		return
+	}
+
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Integration Tests
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+func TestIntegrationRetrieveRouterPortFromConfigServiceAfterRouterStartup(t *testing.T) {
```

Comment:
Won't this test be more comprehensive if you first add the port to the config service using the "addPortToConfigService" method your created and then get it back from the config service to verify that it was correctly added?